conferences/2019/general.json

```diff
@@ -186,5 +186,16 @@
     "twitter": "@CodemotionWorld",
     "cfpUrl": "https://tickets.codemotion.com/rome2019-codemotionworld-com-call-for-papers-guidelines",
     "cfpEndDate": "2018-12-12"
+  },
+  {
+    "name": "!!Con West 2019",
+    "url": "http://bangbangcon.com/west/",
+    "startDate": "2019-02-23",
+    "endDate": "2019-02-24",
+    "city": "Santa Cruz, California",
```

Comment:
Can you put « CA » instead of California to follow other city name we have in the US?

Reply:
done